web/static/js/app.js

```diff
@@ -11,41 +11,46 @@
 //
 // If you no longer want to use a dependency, remember
 // to also remove its path from "config.paths.watched".
-import "deps/phoenix_html/web/static/js/phoenix_html"
+import React from 'react'
+import ReactDOM from 'react-dom'
+
+import '../../../deps/phoenix_html/web/static/js/phoenix_html'
```

Comment:
Changed these to relative URLs so eslint could verify the path. Not sure how phoenix is injecting `deps/` into node's load path, I'd like to make that explicit so we can revert this.